src/traces/table/constants.js

```diff
@@ -0,0 +1,33 @@
+/**
+* Copyright 2012-2017, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+'use strict';
+
+module.exports = {
+    maxDimensionCount: 60,
+    overdrag: 45,
+    cellPad: 8,
+    latexCheck: /^\$.*\$$/,
```

Comment:
That's probably something we have already in https://github.com/plotly/plotly.js/blob/master/src/lib/svg_text_utils.js

Reply:
Will check!